packages/core/test/css-vars.spec.ts

```diff
@@ -58,6 +68,39 @@ describe('css custom-properties (vars)', () => {
             });
         });
 
+        it('global (unscoped) declarations inside rules', () => {
+            const { cssVars, diagnostics } = processSource(
+                `
+                .root {
+                    --a: blue;
+                    stGlobal(--b): red;
+
+                    color: var(stGlobal(--c), red);
+                }
+            `,
+                { from: 'path/to/style.css' }
+            );
+
+            expect(diagnostics.reports.length, 'no reports').to.eql(0);
+            expect(cssVars).to.eql({
```

Comment:
what about the case where you use the same var both in `stGlobal` and without it?
it might be better just to prevent transformation, but not add anything to the metadata when simply using it like in this case.

Reply:
What makes this argument stronger that this is a variable usage and not definition.